test/browser/devtools.js

```diff
@@ -193,7 +193,7 @@ describe_('React Developer Tools integration', () => {
 		node._component.setState({count: 4});
 		node._component.forceUpdate();
 
-		expect(renderer.Reconciler.mountComponent).to.have.been.called.twice;
+		expect(renderer.Reconciler.mountComponent).to.have.been.called.callCount(5);
```

Comment:
As mentioned: `.twice` was `undefined` in master. That's why nothing was checked here. The actual call count is 5. This likely is a bug.

Reply:
@marvinhagemeister I'd prefer to skip this test but keeping desired output rather than hardcoding the actual result..

Reply:
@k1r0s You're right, that's the better way to go. Pushed a new commit that skips this test and adds a todo comment.